openslides/asgi.py

```diff
@@ -7,3 +7,5 @@
 setup_django_settings_module()
 
 channel_layer = get_channel_layer()
+# Use native twisted mode
```

Comment:
Please add a newline before the comment.

Reply:
Done